packages/tools/viewer/src/viewer.ts

```diff
@@ -1063,7 +1081,7 @@ export class Viewer implements IDisposable {
                 getWorldBounds: (animationIndex: number): Nullable<ViewerBoundingInfo> => {
                     let worldBounds: Nullable<ViewerBoundingInfo> = cachedWorldBounds[animationIndex];
                     if (!worldBounds) {
-                        worldBounds = computeBoundingInfos(assetContainer, assetContainer.animationGroups[animationIndex]);
+                        worldBounds = computeBoundingInfos(assetContainer.meshes, assetContainer.animationGroups[animationIndex]);

```

Comment:
This could be:
`worldBounds = computeModelsBoundingInfos(model);`
and then we could remove `computeBoundingInfos`, right?